Formula/m/mesa.rb

```diff
@@ -154,6 +153,9 @@ def install
         -Dvulkan-drivers=amd,intel,intel_hasvk,swrast,virtio
         -Dvulkan-layers=device-select,intel-nullhw,overlay
       ]
+      if Hardware::CPU.intel?
+        args << "-Dgallium-drivers=r300,r600,radeonsi,nouveau,virgl,svga,swrast,i915,iris,crocus,zink"
```

Comment:
`virgl` and `svga` are software drivers for VMs, so they aren't hardware-specific and may be used on other platforms. And `swrast` is a pure software, i.e. CPU driver, so should also be enabled unconditional.